app/src/main/java/net/bible/android/control/page/CurrentMyNotePage.kt

```diff
@@ -123,6 +133,11 @@ class CurrentMyNotePage internal constructor(
         doSetKey(key)
 	}
 
+    @JavascriptInterface
+    override fun save(text: String) {
+        myNoteControl.saveMyNoteText(text)
+    }
+
```

Comment:
Perhaps better that we have all js interface functions in net.bible.android.view.activity.page.BibleJavascriptInterface. Or what do you think?

Reply:
I only discovered later that there was a JavaScript interface already, I'll move it over to there.